doc/src/manual/control-flow.md

```diff
@@ -712,6 +712,18 @@ Stacktrace:
 
 ### The `try/catch` statement
 
+A simple `try/catch` example below shows an `Exception` being raised. For example, calling an array a[] at index 0 would raise a BoundsError() `Exception` since Julia indexes from 1 not 0:
+
+```jldoctest
+julia> f(x) = 
+       a = Int64[]
+       try
+           a[0]
+       catch e
+           println(e)
+       end
+```
+
```

Comment:
This example looks great to me!

When you write a doctest with the keyword `jldoctest`, you're creating an opportunity for Documenter to check that the example you've provided is still relevant/correct, i.e. that Julia produces the expected output. So, you want to include not only code, but the output that you get from running that code at the REPL. For example, the doctest you've created should pass if you change it to

```jldoctest
julia> f(x) = 
       a = Int64[]
       try
           a[0]
       catch e
           println(e)
       end
f (generic function with 1 method)
```

However, I don't think there's really any reason to wrap the above example in a function. If you just run the code you've placed inside the function at the REPL, you'll be able to demonstrate the behavior of try/catch directly:

```jldoctest
julia> a = Int64[]
       try
           a[0]
       catch e
           println(e)
       end
BoundsError(Int64[], (0,))
```

Does that make sense?